sympy/plotting/textplot.py

```diff
@@ -79,6 +79,9 @@ def textplot_str(expr, a, b, W=55, H=18):
                 mi, ma = -1, 1
     else:
         mi, ma = -1, 1
+    if ma-mi > 1.75:
```

Comment:
I would recommend the nearest appropriate multiple of 1, 2 or 5. For example, if limits were 3.9 and 4.6 this would change them to 3 and 5 whereas 3.5 and 5.0 would be more fair.

Reply:
For such a small range (3.9, 4.6)
4.6 - 3.9 = 0.7
As 0.7 < 1.75. No rounding off will take place here. But I understand what you mean. I think a better alternative is to rounding off maximum and minimum according to (maximum-minimum). This could be done based on the most significant digit in the range.